### PR TITLE
Update script command on conda skeleton pypi

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Update script command on conda skeleton pypi to use `{{ PYTHON }} -m pip install . -vv`

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
Related to #3716 

cc: @CJ-Wright

File: conda_build/skeletons/pypi.py [hunk 1]
```diff
@@ -339,7 +339,9 @@ def skeletonize(packages, output_dir=".", version=None, recursive=False,
             ordered_recipe['source']['sha256'] = d['hash_value']
 
             if d['entry_points']:
-                ordered_recipe['build']['entry_points'] = d['entry_points']
+                ordered_recipe['build']['script'] = (
```
Comment (marcelotrevisani) [inline conda_build/skeletons/pypi.py]: geez. copy paste on the wrong place,

**Comment by ocefpaf** [comment]
Thanks! This will match closely to what we are using in conda-forge and will reduce the number of edits :smile: 

@marcelotrevisani, on a related note, conda-skeleton is "quoting" all the lines and that breaks the bot updates. I guess that it would be nice to have at least the checksum unquoted.

**Comment by marcelotrevisani** [comment]
> Thanks! This will match closely to what we are using in conda-forge and will reduce the number of edits smile
> 
> @marcelotrevisani, on a related note, conda-skeleton is "quoting" all the lines and that breaks the bot updates. I guess that it would be nice to have at least the checksum unquoted.

Sure! I can take a look at it as well!
Could you just open a new issue (just to have some track and to not forgot) for it an give me an example just to be sure that is what I am thinking? haha

**Pull request merged**